build/basic-template.html

```diff
@@ -1,133 +0,0 @@
-                        </td>
-                      </tr>
-                    </table>
-                  </td>
-                </tr>
-
-                <tr style="border-spacing: 0; margin: 0; padding: 0;">
-  <td style="border-spacing: 0; margin: 0; padding: 0;">
-    <table class="templateFooter" style="border-spacing: 0; margin: 0; margin-top: 20px; padding: 0; width: 100%;">
-      <tr style="border-spacing: 0; margin: 0; padding: 0;">
-        <td class="footerContent" style="border-spacing: 0; color: #707070; font-size: 12px; line-height: 125%; margin: 0; padding: 0;">
-          <table style="border-spacing: 0; margin: 0; padding: 0; width: 100%;">
-            <tr style="border-spacing: 0; margin: 0; padding: 0;">
-              <td class="social" style="border-bottom: 1px solid #eee; border-spacing: 0; border-top: 1px solid #eee; font-size: 14px; margin: 0; padding: 25px 0; text-align: center;">
-                <a href="#" style="color: #348eda; line-height: 1.5
-  font-size: 14px; margin: 0; padding: 0; text-decoration: none;">Twitter</a> ·
-                <a href="#" style="color: #348eda; line-height: 1.5
-  font-size: 14px; margin: 0; padding: 0; text-decoration: none;">Facebook</a>
-              </td>
-            </tr>
-          </table>
-          <table style="border-spacing: 0; margin: 0; padding: 0; width: 100%;">
-            <tr style="border-spacing: 0; margin: 0; padding: 0;">
-              <td class="utility" style="border-spacing: 0; color: #999; font-size: 12px; margin: 0; padding: 20px 0; text-align: center;">
-                <div style="margin: 0; padding: 0;">
-                  <a href="#" style="color: #999; font-size: 12px; margin: 0; padding: 0; text-decoration: none;">Unsubscribe</a> — <a href="#" style="color: #999; font-size: 12px; margin: 0; padding: 0; text-decoration: none;">View in Browser</a>
-                </div>
-              </td>
-            </tr>
-          </table>
-        </td>
-      </tr>
-    </table>
-  </td>
-</tr>
-
-</table>
-
-<table class="templateAddress" style="border-spacing: 0; color: #999; font-size: 12px; line-height: 125%; margin: 0; padding: 20px; text-align: center; width: 100%;">
-  <tr style="border-spacing: 0; margin: 0; padding: 0;">
-    <td style="border-spacing: 0; margin: 0; padding: 0;">
-      <br style="margin: 0; padding: 0;">
-        This email template was prepared for you by your friends at <a href="https://kontist.com/" style="color: #999; font-size: 12px; margin: 0; padding: 0; text-decoration: underline;">Kontist</a>.<br style="margin: 0; padding: 0;">
-        Logo and image from <a href="http://mailchimp.com/" style="color: #999; font-size: 12px; margin: 0; padding: 0; text-decoration: underline;">Mailchimp</a>.
-      <br style="margin: 0; padding: 0;">
-    </td>
-  </tr>
-</table>
-
-
-            </td>
-          </tr>
-        </table>
-
-      </td>
-    </tr>
-  </table>
-</body>
-</html>
```

Comment:
ahhh well it's the build template, nevermind, I may wanna put that in the .gitignore anyway